site/reference/README.md

```diff
@@ -16,7 +16,7 @@ description: >
 
 <!--mdtogo:Long-->
 
-Usage: kpt \<group> \<command> \<positional args> [PKG_PATH] [flags]
+Usage: `kpt <group> <command> <positional args> [PKG_PATH] [flags]`
```

Comment:
use block quotes (triple quoted)

Reply:
Tried 

Usage: ```kpt <group> <command> <positional args> [PKG_PATH] [flags]```
---
Usage: 

```
kpt <group> <command> <positional args> [PKG_PATH] [flags]
```
---
```
kpt <group> <command> <positional args> [PKG_PATH] [flags]
```

Third one is more cleaner so went with it.

Reply:
![image](https://user-images.githubusercontent.com/58999035/118020002-7c39aa00-b30e-11eb-9ed0-6efb98ea1331.png)